lib/timetrap/cli.rb

```diff
@@ -443,7 +447,12 @@ def yesterday
     end
 
     def week
-      args['-s'] = Date.today.wday == 1 ? Date.today.to_s : Date.parse(Chronic.parse(%q(last monday)).to_s).to_s
+      d = Chronic.parse( args['-s'] || Date.today )
```

Comment:
Actually, now that I look at this, I should probably remove allowing the `-s` argument here, otherwise it will take everything from the `-s` date until today, which could be more than a week. 

Alternatively, changing how the end date argument in line 455 is set could be more complex, if you want to allow for someone to set a start date for week. Then timetrap would go back to the last day that match `week_start` configuration and show all entries from that date and 6 days following it. The date specified in the user-inputted value for `t week -s` would be contained in this query. This would match the behavior for `t month`, which allows for pulling up previous months.